### PR TITLE
Update some requirements and link to github

### PR DESCRIPTION
**Pull request by TheOriginalSoni**

### CODE REVIEW

File: web_static/usage.html [hunk 1]
```diff
@@ -370,7 +370,5 @@ <h2><a name="how">How It Works (and Why It's So Slow Sometimes)</a></h2>
 using the service.</p>
 
 <hr>
-<address>updated March 24, 2009 - egnor<script>
-document.write("@");
-</script>ofb.net</address>
+<address>updated March 24, 2009 - egnor<script>document.write("@");</script>ofb.net</address>
```
Comment (egnor) [inline web_static/usage.html]: You can just put in my real email, no need for any obfuscation, I'm long past trying to hide it from the spammers lol

File: dev_requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 conan==2.1.0
 cmake==3.28.3
+daemon==1.2
```
Comment (egnor) [inline dev_requirements.txt]: This probably won't have the desired effect for `cgi-slack.py`, which would be run directly by the web server outside of whatever virtualenv is used here. This really is "dev requirements", i.e. things needed for build.

Maybe it's best to just drop `cgi-slack.py` entirely. We (Left Out) used it for a slackbot for a while, but... (a) everyone's moving to Discord and (b) even on Slack that got basically no usage. I think a generic REST API of some kind would make more sense (I think a lot of people just scrape the HTML currently?)

File: cgi_scripts/cgi-search.py [hunk 3]
```diff
@@ -65,9 +65,9 @@
 <ul>
 <li><a href="usage.html">Usage guide</a>: usage tips,
 worked examples, why it's slow.
-<li><a href="https://github.com/egnor/nutrimatic">Source code</a>:
-badly documented, but see the <a
-href="https://github.com/egnor/nutrimatic/blob/master/README">README</a>.
+<li><a href="https://github.com/PuzzleTechHub/nutrimatic">Source code</a>:
+not completely documented, but see the <a
+href="https://github.com/PuzzleTechHub/nutrimatic/blob/master/README.md">README</a>.
```
Comment (egnor) [inline cgi_scripts/cgi-search.py:70]: Thanks!

**Review by egnor — CHANGES_REQUESTED** [review]

File: dev_requirements.txt (hunk 2 above)
Comment (TheOriginalSoni) [reply dev_requirements.txt]: I removed it from dev_requirements again. I dont know if just removing cgi-slack.py will have any other effects, so not removing it myself yet.

**Review by egnor — APPROVED** [review]

**Pull request merged**